### PR TITLE
Bridge dojot v2k

### PR DESCRIPTION
**Pull request by muhamedcpqd**
* **Please check if the PR fulfills these requirements**
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Refactor

* **What is the current behavior?** (You can also link to an open issue here)
the service was not compliance with dojot

* **What is the new behavior (if this is a feature change)?**
the service is compliance with dojot env

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
no

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
dojot/dojot#1314

* **Other information**:

### CODE REVIEW

File: v2k-bridge/src/AgentMessenger.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            mqttClient.subscribe(this.config.mqtt.subscribeTopic);
+        }).catch(() => {
+            logger.error("An error occurred while initializing the IoTAgent. Bailing out!", TAG);
+        });
+
+    }
+
+    sendMessage(topic, message) {
+        try {
+            logger.info(`Message published`, TAG)
```
Comment (jonaphael) [inline v2k-bridge/src/AgentMessenger.js]: set as debug

File: v2k-bridge/src/AgentMessenger.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        this.initialized = false;
+        this.config = config || defaultConfig;
+
+        /* set log level */
+        logger.setLevel(this.config.app.mqtt_log_level);
+
+        this.iotagent = new IoTAgent();
+    }
+
+    init(mqttClient) {
```
Comment (jonaphael) [inline v2k-bridge/src/AgentMessenger.js]: check if mqttClient param is not undefined, also if possible the possibility if it is connected.

File: v2k-bridge/.eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,294 @@
+{
```
Comment (mprevide) [inline v2k-bridge/.eslintrc.json]: A ideia é usar o da airbnb :)
Da uma olhada serviço do prometheus

File: ansible-dojot/roles/v2k-bridge/vars/main.yaml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# Variable values for the apigw role
+
+dojot_apigw_version: v0.2.1-static
```
Comment (mprevide) [inline ansible-dojot/roles/v2k-bridge/vars/main.yaml]: Não deve ser usado

File: ansible-dojot/roles/v2k-bridge/defaults/main.yaml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+# This files has the default values for dojot API Gateway variables
+
+dojot_internal_http_port: 8000
```
Comment (mprevide) [inline ansible-dojot/roles/v2k-bridge/defaults/main.yaml]: Acho que tb não sao usados

File: v2k-bridge/.eslintrc.json (hunk 3 above)
Comment (muhamedcpqd) [reply v2k-bridge/.eslintrc.json]: ok.

File: v2k-bridge/package.json [hunk 6]
```diff
@@ -27,6 +27,12 @@
     ]
   },
   "devDependencies": {
+    "eslint": "^6.1.0",
```
Comment (mprevide) [inline v2k-bridge/package.json]: Acredito que só seja necessário:
 ``  "eslint": "^6.8.0",
    "eslint-config-airbnb-base": "^14.0.0",
    "eslint-plugin-import": "^2.19.1", ``

O restante é relacionado a react. 
Inclusive vou adicionar no serviço que eu fiz.

**Pull request merged**